code/datums/chatmessage.dm

```diff
@@ -44,10 +53,6 @@
 	var/eol_complete
 	/// Contains the approximate amount of lines for height decay
 	var/approx_lines
-	/// Contains the reference to the next chatmessage in the bucket, used by runechat subsystem
-	var/datum/chatmessage/next
-	/// Contains the reference to the previous chatmessage in the bucket, used by runechat subsystem
-	var/datum/chatmessage/prev
```

Comment:
Where'd these go? were they just unused?

Reply:
They were unused

Reply:
they have been unused since SSrunechat was refactored to be a subtype of SStimer